Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -935,6 +968,16 @@ theorem mem_closure_one_iff_norm {x : E} : x ∈ closure ({1} : Set E) ↔ ‖x
 theorem closure_one_eq : closure ({1} : Set E) = { x | ‖x‖ = 0 } :=
   Set.ext fun _x => mem_closure_one_iff_norm
 
+section Instances
+
+instance [SeminormedGroup E] : ContinuousENorm E where
+  continuous_enorm := continuous_enorm'--ENNReal.continuous_coe.comp continuous_nnnorm'
```

Comment:
```suggestion
@[to_additive]
instance SeminormedGroup.toContinuousENorm [SeminormedGroup E] : ContinuousENorm E where
  continuous_enorm := continuous_enorm'
```
tested locally

Reply:
Of course, the lemma `continuous_enorm'` should be removed, and there should be a single `continuous_enorm` lemma that assumes the `ContinuousENorm` class.

Reply:
Indeed! Changed, hopefully.